etc/argus.conf

```diff
@@ -129,6 +129,12 @@ recipe = argus.recipes.cloud.windows:CloudbaseinitScriptRecipe
 userdata = argus.windows.multipart_userdata
 
 
+[scenario_sethostname_userdata : base_smoke_windows]
+test_classes = argus.tests.cloud.smoke:TestSetHostname,
+               argus.tests.cloud.smoke:TestNoError
+userdata = argus.windows.multipart_userdata
```

Comment:
You have to remove TestSetHostname from the multipart scenario, as well as removing that content from the multipart_userdata file.